### PR TITLE
add switch..case condition

### PR DESCRIPTION
**Pull request by zarszz**
I have added content for switch..case condition, 
if found a weird or wrong thing, please notify me, because I newly to flutter

### CODE REVIEW

File: lib/ifelse.dart [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Ifelse extends StatelessWidget{
    Widget build(BuildContext context){
     return Scaffold(
       appBar: AppBar(
-        title: Text("if else"),
+        title: Text("if..else                       "),
```
Comment (harry-hov) [inline lib/ifelse.dart:9]: Why blank space after if..else ?

**Review by harry-hov — APPROVED** [review]
Looks good to me. I'll merge this soon.

**Comment by harry-hov** [comment]
Also try to target only one issue in a single PR @zarszz.
i.e. You can make a another PR for "if..else" fixes.

**Review by udaymewada — APPROVED** [review]

File: lib/ifelse.dart (hunk 1 above)
Comment (zarszz) [reply lib/ifelse.dart:9]: umm, I think .. in can representation a else if

Comment (zarszz) [reply lib/ifelse.dart:9]: thanks for correction

**Pull request merged**